react-web/src/pages/certification/Certification.tsx

```diff
@@ -247,6 +248,20 @@ const Certification = () => {
       handleErrorScenario
   )
 
+  const [render, setRendered] = useState(false);
+
+  useEffect(() => {
+    // Initial load of subscribed features
+    if (subscribedFeatures !== null) {
+      setRendered(true);
+    }
+  }, [subscribedFeatures]);
```

Comment:
Because, right below that in https://github.com/input-output-hk/dapps-certification/pull/78/files/f9d34cd4efaa02dc877a05eea2a42198339fa0cf#diff-b3044d4e1fb9e91f06320855a0883bdb0f49c3fee056b2ac3cc8c7096d9c3f82R260-R264 I am trying to block the page with Loader until the subscribedFeatures are fetched, and for the render to update it needs to be a state. Correct me if I am wrong.